src/hexo/themes/sonarwhal/layout/partials/contributors.hbs

```diff
@@ -1,27 +1,25 @@
-<main class="container">
-    <article class="content" aria-labelledby="page-heading">
-        <nav class="container breadcrumb" aria-label="breadcrumbs">
-            <ul>
-                <li><a href="/">Home</a></li>
-                <li><a href="../">About</a></li>
-                <li>Contributors</li>
-            </ul>
-        </nav>
-        <header class="page-intro">
-            <h1 id="page-heading">{{page.title}}</h1>
-        </header>
-        <section class="section">
-            <div class="layout layout--quarters">
-            {{#each devs}}
-                <div class="module module--contributors">
-                    <a {{#if website}} href="{{website}}"{{else}}href="#"{{/if}}>
-                        <img src="{{avatar}}?s=100" width="100" alt="{{name}}">
-                    </a>
-                    <p class="subtitle">{{name}}</p>
-                    <p class="caption--alt">{{company}}</p>
-                </div>
-            {{/each}}
+<article class="content" aria-labelledby="page-heading">
+    <nav class="container breadcrumb" aria-label="breadcrumbs">
```

Comment:
The `breadcrumb` can be abstracted and put in `about.hbs` so that it's shared throughout the about pages. I can add that change after you are done if that is the design 😊 .

Reply:
I will let you know when I'm done with the update.